include/oneapi/dpl/pstl/parallel_backend_tbb.h

```diff
@@ -1248,25 +1249,26 @@ operator()(__task* __self)
         return nullptr;
     }
 
-    _RandomAccessIterator1 __xm;
-    _RandomAccessIterator2 __ym;
+    ::std::optional<_RandomAccessIterator1> __xm;
```

Comment:
As far as I found, we have another place like this for OpenMP backend.

Reply:
Yes, OMP backend should be fixed as well. Thanks.

Reply:
Fixed

Reply:
ok